src/transformers/models/wav2vec2/modeling_tf_wav2vec2.py

```diff
@@ -268,7 +268,7 @@ def _compute_mask_indices(
             f" `sequence_length`: {sequence_length}`"
         )
     # compute number of masked spans in batch
-    num_masked_spans = int(mask_prob * sequence_length / mask_length + tf.random.uniform((1,)))
+    num_masked_spans = int(mask_prob * sequence_length / mask_length + np.random.uniform(size=(1,)))
```

Comment:
Same as with hubert - changing to numpy call to enable int cast in graph mode (and allow downstream computations that assume int).